.zuul.yaml

```diff
@@ -4,20 +4,11 @@
         - "thoth-coala"
         - "thoth-pytest"
     gate:
-      queue: "CHANGEME"
```

Comment:
I think this one is required.

Reply:
Does the `.zuul.yaml` look right now? I got the one we have in thoth-common